src/main/java/cz/cvut/kbss/study/service/SystemInitializer.java

```diff
@@ -52,7 +52,7 @@ private void addDefaultAdministrator() {
             admin.setPassword("5y5t3mAdm1n.");
             admin.setInstitution(institutionService.findByName(INSTITUTION_NAME));
             admin.setIsInvited(true);
-            admin.getTypes().add(Vocabulary.s_c_administrator);
+            admin.setRoleGroup(Vocabulary.s_i_operator_admin_role_group);
```

Comment:
This will not work. We would have to persist RoleGroup as well in case we wont this to work